scripts/publish-instance-metrics/on-start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+# OVERVIEW
+# This script publishes the system-level metrics from the Notebook instance to Cloudwatch.
+
```

Comment:
This script has some dependencies that should be documented.

1. Internet access to fetch the example config
2. The Notebook Instance role will need permissions to `cloudwatch:PutMetricData` to publish these metrics
3. Since Cloudwatch Metrics are charged on a per metric level, consider linking to the [pricing page](https://aws.amazon.com/cloudwatch/pricing/) so that users are aware of the expected costs